### ISSUE TITLE
Vector test failing on powerpc

### ISSUE THREAD

**Issue by Tachi107** [zyantific/zycore-c#59]
Hi, it seems that the `VectorTest.Destructor` test is failing on Debian's powerpc and powerpc64 architectures :/

I have not looked into the issue, but I guess that this might be related to ppc's big endianness.

You can find the logs for both here:

- <https://buildd.debian.org/status/logs.php?pkg=zycore-c&arch=powerpc>
- <https://buildd.debian.org/status/logs.php?pkg=zycore-c&arch=ppc64>

**Comment by athre0z** [comment]
Yeah, big-endian is currently completely unsupported and this would require fixes all over the code-base.

**Comment by Tachi107** [comment]
Oh ok, thanks for the fast reply! powerpc isn't an "official" Debian architecture, so this isn't really a big deal. Feel free to close this, or to keep it open as a kind of todo item :)

**Comment by athre0z** [comment]
Cool. Let's leave this open as a low priority TODO item and as a reference for other people who are wondering about the BE support status. I adjusted the title to improve discoverability.

**Comment by mochaaP** [comment]
Fixed in #87
